src/components/Providers.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+import { MovieProviders } from '../types/tmdb';
+import { getMovieProviders } from '../helpers/getMovieUtils';
+
+interface ProviderProps { ID: number | null }
+/**
+ * Component to display streaming services logos for a given movie or show. Accepts an ID and
+ * @returns {JSX.Element}
+ */
+export default function Providers(props: ProviderProps): JSX.Element {
+    const [providers, setProviders] = useState<MovieProviders>();
+
+    useEffect(() => {
+        const handler = async () => {
+            if (props.ID) {
+                const data = await getMovieProviders(props.ID);
+                setProviders(data);
+            }
+        };
+        handler();
+    }, []);
+
+    return (
+        <div className='flex justify-center'>
+            {providers?.results.US !== undefined ? (
```

Comment:
It would be cool to pull the country code from the user model in the future so we can have this be more dynamic. Nothing set up right now, just an idea I figured I would note here :)